docs/source/using/profiles.adoc

```diff
@@ -63,4 +82,41 @@ To delete a profile, run:
 $ minishift profile delete PROFILE_NAME
 ----
 
-NOTE: The default profile 'minishift'  can not be deleted.
+NOTE: The default profile *minishift* cannot be deleted.
+
+[[example-workflows-profiles]]
+== Example Workflows
+
+To create a new profile *foo* with specific configurations, run:
+
+----
+$ minishift profile set foo
+$ minishift config set memory 4GB
+$ minishift config set cpus 2
+$ minishift addon enable anyuid
+$ minishift start
+----
+
+Alternate method to create the profile *foo*:
+
+----
+$ minishift --profile foo config set memory 4GB
+$ minishift --profile foo config set cpus 2
+$ minishift --profile foo addon enable anyuid
+$ minishift --profile foo minishift start
+----
+
+Note: Once you have started multiple profiles, you should use the xref:switching-profiles[`profile set`] command to switch between them.
+
+[[global-config-profiles]]
+== Global Configuartion For profiles
+
+Currently, we cannot set any global configuration for profiles.
+If you want to apply a common configuration to all your profiles then you need to apply the settings for each profile separately.
+As a workaround, xref:../using/basic-usage.adoc#environment-variables[environment variables] can be used.
+
+To set 8GB memory for all profiles on Linux/macOS, run:
```

Comment:
"To allocate 8GB of memory for all profiles use the MINISHIFT_MEMOR environment variable. How you set the environment variable is OS specific. For Linux or macOS run:"

Alternatively give both options Linux/macOS and Windows.

Currently it sounds a bit like it only works for Linux/macOS.

Reply:
I do not think I should give an example here. The right thing to do is to point the users to the right documentation for environment variables.

Reply:
@LalatenduMohanty Agreed, the example seems unnecessary. We should simply guide the reader to the appropriate information.

@hferentschik @LalatenduMohanty Considering the note about elaboration made, I think it makes sense to elaborate on the use of environment variables within their own documentation. In my opinion, we shouldn't need to go into any _detail_ regarding environment variables in the documentation for profiles.